.github/scripts/Rename_Photos

```diff
@@ -0,0 +1,26 @@
+#This script renames photos from BHVP_PH_CetaitParis_DP_1538_01_015 and BHVP_PH_CetaitParis_DP_1538_01_016.jpg to 16_slide.jpg and 16_photo.jpg
+
+import math
+import os
+import csv
+
+folder = r"/Users/ismailassafi/Documents/GitHub/paris_1970/assets/images/photos/1538/"
```

Comment:
This path shouldn't be hardcoded! We will probably have a different a different path to input

Reply:
Take a look at @czheng10's scripts in this folder and how they deal with this: https://github.com/dhmit/paris_1970/blob/main/backend/scripts/imageconversion.py

Reply:
The function should take in a source dir and a destination dir as arguments.